packages/ag-charts-community/src/chart/series/cartesian/boxPlotOptions.ts

```diff
@@ -32,14 +22,59 @@ export interface AgBoxPlotSeriesOptions<DatumType = any>
     q3Name?: string;
     maxKey?: string;
     maxName?: string;
+}
+
+export interface AgBoxPlotCapOptions {
+    lengthRatio?: Ratio;
+}
 
-    /** Configuration for the labels shown on columns. */
-    // label?: AgColumnSeriesLabelOptions;
+export type AgBoxPlotWhiskerOptions = StrokeOptions & LineDashOptions;
+
+export interface BoxPlotNodeDatum
+    extends Omit<CartesianSeriesNodeDatum, 'yKey'>,
```

Comment:
Nit: Maybe move the `Omit<...>` to the end, as at first glance I was trying to parse whether the following extends clauses were parameters to that `Omit`?

Reply:
sure, if you think it helps readability